stdlib/public/Concurrency/CFExecutor.cpp

```diff
@@ -0,0 +1,28 @@
+//===--- CFExecutor.cpp ----------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+#include "swift/Runtime/Concurrency.h"
+
+#if !defined(_WIN32) && !defined(__wasi__) && __has_include(<dlfcn.h>)
+#include <dlfcn.h>
+#endif
```

Comment:
```suggestion
#include <dlfcn.h>
```

This seems like we shouldn't bother with the guards as this is now Darwin specific.